frontend/src/console/components/NoticeBoard/NoticeBoard.tsx

```diff
@@ -194,15 +194,30 @@ const NoticeBoardComponent: React.FC<Props> = (props: Props) => {
     </p>
   );
 
+  const staticExplanation = (
+    <div className="explanation-container">
+      <p>
+        <WrappedMessage id="explanation" messages={messages} />
+        <br />
+        <br />
+        <WrappedMessage id="blogpost_text" messages={messages} />
+        <a href="https://opencraft.com/blog">
+          <WrappedMessage id="blogpost_link" messages={messages} />
+        </a>
```

Comment:
Let's use the following approach (along with the suggestion from `NoticeBoard/displayMessages.ts`:
```js
// TODO: Point the link to a specific blog post, once it's created.
<WrappedMessage
  id="blogpost_text"
  messages={messages}
  values={{
    link: (text: string) => (
      <a href="https://opencraft.com/blog">{text}</a>
    )
  }}
/>
```

Reply:
I'll keep this one in mind! Thanks! Applied.